sql/moz-fx-data-shared-prod/firefox_desktop/newtab_live/metadata.yaml

```diff
@@ -7,7 +7,10 @@ description: |-
   document type.
 
   Clustering fields: `submission_timestamp`
+labels:
+  authorized: true
```

Comment:
I was incorrect about the configuration here. It turns out authorizations aren't needed if we're going with https://github.com/mozilla-services/cloudops-infra/blob/dcede32b67cb424bb7475dbd4f8afd845aa098e5/projects/data-shared/tf/prod/envs/prod/bigquery-new/namespaces.auto.tfvars.json#L432

```suggestion
```